akka-docs/src/main/paradox/scala/distributed-pub-sub.md

```diff
@@ -109,7 +129,7 @@ cluster aware router where the routees dynamically can register themselves.
 The message will be delivered to one recipient with a matching path, if any such
 exists in the registry. If several entries match the path because it has been registered
 on several nodes the message will be sent via the supplied `RoutingLogic` (default random)
-to one destination. The sender() of the message can specify that local affinity is preferred,
+to one destination. The sender of the message can specify that local affinity is preferred,
```

Comment:
This is the conceptual sender of the message, not the `sender()` method of an actor.